docs/sources/style-guide/voice-tone-guidelines/index.md

```diff
@@ -29,7 +29,9 @@ Write in a natural and conversational style with language that reflects a positi
 
 A casual tone is accepted in UI copy and supporting material compared to traditional business writing.
 
-As long as they allow for shorter sentences, you can use the words _and_, _but_, and _so_ in moderation.
+As long as they allow for shorter sentences, you can use the words _and_, _but_, and _so_ in moderation. 
+
+Use contractions like _isn't_ for _is not_ to help to convey an informal tone. But [be careful](https://www.grammarly.com/blog/contractions/).
```

Comment:
```suggestion
Use contractions such as _isn't_ for _is not_ because they are easier to understand and convey a less formal tone. For detailed information about contractions, see [What Are Contractions in Writing?](https://www.grammarly.com/blog/contractions/) by Matt Ellis.
```